spec/std/isa/inst/C/c.lw.yaml

```
@@ -13,9 +13,10 @@ description: |
   to the base address in register xs1.
   It expands to `lw` `xd, offset(xs1)`.
 definedBy:
-  anyOf:
-    - C
-    - Zca
+  extension:
+    anyOf:
+      - name: C
+      - name: Zca
```

In `c.lui.yaml`, this shrank to just:

```suggestion
    - name: Zca
```
which seems optimal. Shall we implement that simplification everywhere this appears?